src/linux_android_with_fallback.rs

```diff
@@ -31,3 +31,65 @@ fn is_getrandom_available() -> bool {
         true
     }
 }
+
+// Polls /dev/random to make sure it is ok to read from /dev/urandom.
```

Comment:
As explained in the first commit, This code is only used on Linux/Android. With the other changes in this PR, `DropGuard` also became Linux/Android-specific. Instead of adding more `#[cfg(...)]` I just moved it all.